core/prometheus_exporter.go

```diff
@@ -0,0 +1,131 @@
+package core
+
+import (
+	"fmt"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/pkg/errors"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	namespace = "gorb" // For Prometheus metrics.
+)
+
+var (
+	serviceHealth = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: namespace,
+		Name:      "service_health",
+		Help:      "Health of the load balancer service",
+	}, []string{"name", "host", "port", "protocol", "method", "persistent"})
```

Comment:
Fair enough.
I was thinking more along the lines that method and persistence would differentiate the characteristics of the metrics being measured but if you don't think that's applicable I can remove them.